express-delete/server.js

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+app.get('/api/grades', (req, res) => {
+  const arr = [];
+  for (const key in grades) {
+    arr.push(grades[key]);
+  }
+  res.json(arr);
+});
+
+// const id = process.argv[3].slice(26)
+app.delete('/api/grades/12', (req, res) => {
```

Comment:
These work for the cases you've code, but it's not generalizable. Get the ID from the parameters (how to do that is in the doc referenced in the instructions) and use that.